lib/dockerize_rails/docker_commands/docker_build.rb

```diff
@@ -31,10 +41,15 @@ def self.build_mysql
 
       def self.build_postgres
         if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'postgresql'
+          if Docker::Image.exist? DockerHelpers.get_name(:postgres, :image)
+            puts " ==> Image >#{DockerHelpers.get_name(:postgres, :image)}< already exists".green
+            return 0
+          end
           build_docker_image(
-            "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::PG_DIRECTORY_NAME}/Dockerfile",
-            "#{DRConfig.application_name}_postgres"
+              "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::PG_DIRECTORY_NAME}/Dockerfile",
```

Comment:
__[rubocop]__
* Indent the first parameter one step more than the start of the previous line.

<!-- Comment by [SideCI](https://sideci.com) -->